sip-test-kit/src/main/java/de/ikor/sip/foundation/testkit/workflow/whenphase/routeinvoker/RouteInvoker.java

```diff
@@ -25,10 +26,11 @@ public interface RouteInvoker {
   boolean isApplicable(Endpoint endpoint);
 
   /**
-   * Set endpoint to proper RouteInvoker
+   * Method which classifies which camel components could be suspended during sip batch tests.
```

Comment:
Shouldn't it be:
which camel components **should** be suspended during sip batch tests. (or even **have to be**)

Reply:
than method name should be changed as well

Reply:
Method is completely removed, since there is now different approach to recognise which routes/components are for suspending.